folium/folium.py

```diff
@@ -38,10 +38,9 @@
     ('leaflet_css',
      'https://cdn.jsdelivr.net/npm/leaflet@1.6.0/dist/leaflet.css'),
     ('bootstrap_css',
-     'https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css'),
-    # glyphicons came from Bootstrap 3 and are used for Awesome Markers
-    ('glyphicons_css',
-     'https://cdn.jsdelivr.net/gh/python-visualization/folium/folium/templates/bootstrap_3_glyphicons/bootstrap-3-glyphicons.css'),  # noqa
```

Comment:
The reason for the error is this file doesn't exist in the repo, see: https://github.com/python-visualization/folium/tree/v0.13.0/folium/templates/bootstrap_3_glyphicons/bootstrap-3-glyphicons.css